src/fragments/lib/push-notifications/android/record_notifications/10_handle_notification_received.mdx

```diff
@@ -0,0 +1,100 @@
+<Callout>
+
+You do not need to add these APIs to your app if you want to use Amplify with Amazon Pinpoint. You should use these APIs if you are interested in supporting multiple push providers.
```

Comment:
I found it extremely confusing about what we wanted to clarify with the user here. We are in the register device page and we are mentioning about supporting multiple push providers. 

Since this is part of documentation, we should keep in mind users can land on this page and might not have the information from previous or next pages. We need to be as clear as possible.

Reply:
I had the same initial feedback as well. What do you recommend we change this to instead? We were trying to maintain the same page setup as Swift for example since you do have to set up these APIs when using the service extension. But I do not want to create an sub-optimal experience because we want to maintain consistency.

Reply:
I need to understand what is our message here. Do we want to mention that these pages are not necessary to follow or do we want to mention to only use it in certain situations?